### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by BANCUNGUYE66**
- .gitignore file added
- index.html file added
- style.css file added
- linter.yml added
- README.md added

### CODE REVIEW

**Review by Shisui6 — COMMENTED** [review]
Hi @BANCUNGUYE66 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛠️

You have submitted a project, that;

- Linters are failing. Please go through your code base and check for any errors. Run linter checks on your local machine and fix all warnings that come up
- Does not have a descriptive PR description. Please update your PR description to include a summary of all the changes made in this branch
- Does not contain a readme file. Please make all your future projects use the microverse readme [template](https://github.com/microverseinc/readme-template). When using the microverse template, please fill out and personalize each section to fit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;800&display=swap" rel="stylesheet">       
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+   <header>
+      <nav>
+          <div class="nav_bar">
+             <h2>Aima.B</h2>
```
Comment (mukhammadrizooff) [inline index.html:17]: - [ ] [REQUIRED] Good job so far! The logo should be on the `left`✔️ and has a `link`(a tag)❌ This is for three reasons: `convention`, `semantics`, and `user experience(UX)`. [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info and implement my suggestion please 🙏🏼

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   <main>
+         <section class="homepage">
+            <div class="homepage_container">
+               <h1>I’m Aima Glad to see you! </h1>
+               <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+               </p>
+               <div class="connect">
+                   <a href="#"> LET’S CONNECT</a>
+               </div>
```
Comment (mukhammadrizooff) [inline index.html:33]: - [ ] [REQUIRED] Good job so far! Please add correct design to `<a href="#"> LET’S CONNECT</a>` like a [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=w7hRG2wjWSX2EiST-0) 🙏🏼 

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+               <div class="icon_wrapper">
+                 <img src="images/Vector 1.svg" alt="">
+                 <img src="images/Linkedin icon.svg" alt="">
+                 <img src="images/Vector 3.svg" alt="">
+                 <img src="images/Vector 4.svg" alt="">
+                 <img src="images/Vector 5.svg" alt="">
+               </div>
```
Comment (mukhammadrizooff) [inline index.html:40]: - [ ] [REQUIRED] Good job so far! The `list of icons` should have use a `ul` tag to group them, and `li` per icon; each `icon` should have a `link` (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @BANCUNGUYE66, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  .homepage {
+    width: 100%;
+    height: 100vh;
+    background-image: url(images/header-shapes\ mobile@2x.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: white;
+    padding: 128px 24px;
+    border-bottom-left-radius: 100px;
+  }
```
Comment (mckent05) [inline style.css:43]: **OPTIONAL** - Good job using the `background-image` property to display the image for this section, but you have not properly aligned the background image. Please fix this 🙏 

Your project:
-- Please make sure the marked background image is aligned as shown in the Figma design template 🙏 
hw5hw5h

Figma template:
header portfolio 2

**Review by mckent05 — APPROVED** [review]
Hi @BANCUNGUYE66, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**